server/router/index.js

```diff
@@ -1,9 +1,12 @@
 const express = require('express');
 const { logout } = require('../controllers');
+const { login } = require('../controllers');
```

Comment:
add with logout line two and remove this line
```suggestion

```

Reply:
Done